src/main/java/com/interfaceict/ptrakcer/controllers/ProjectEntityController.java

```diff
@@ -0,0 +1,32 @@
+//
+// Created by htrap19 on 1/8/22
+//
+
+package com.interfaceict.ptrakcer.controllers;
+
+import com.interfaceict.ptrakcer.models.ProjectEntity;
+import com.interfaceict.ptrakcer.servicesimpl.ProjectEntityServiceImpl;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.List;
+
+@RestController
+public class ProjectEntityController
+{
+    @Autowired
+    private ProjectEntityServiceImpl m_Service;
+
+    @GetMapping("/projects")
+    public List<ProjectEntity> getAllProjects() { return m_Service.getAll(); }
```

Comment:
For sample projects we can just return original entity, while for real projects it is a good-practice to use a DTO as return type

Reply:
Yep i thought of using DTO but for now DTO will be useless

Reply:
When entities numbers grow, we have to adapt DTO.